### DISCUSSION TITLE
Display schedule timezone in UI DAG list for timezone-aware DAGs

### DISCUSSION THREAD

**Discussion by mhaalme** [apache/airflow#24009] [Ideas]
**Description**

Display schedule timezone in web UI for timezone-aware DAGs

**Use case / motivation**

We use Airflow to orchestrate tasks in a variety of different systems, some of which contain timezone-dependent data and others work in UTC. Additionally, some DAGs must be run at certain times in our local timezone while others require fixed intervals which should not be affected by daylight savings. Using only UTC or only local is not an option for us and in the future it is likely that some DAGs will depend on a third timezone entirely.

Looking at the cron expressions in the DAG list, there is no way to quickly tell which DAGs are timezone-dependent. This can be confusing, especially for beginners or less frequent Airflow users. For example two DAGs can appear to be scheduled to run in parallel, but their actual start time will be 3 hours apart because one is timezone-aware.

Changing the UI timezone does not affect the cron expressions - I assume that this is intentional and it seems to be reasonable. However, it would be a lot easier to interpret the expressions if the timezone was also displayed next to them.

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by eladkal** [comment]
I don't think Airflow should have timezone column as default in the DAG list page.
Many users use only 1 timezone so this column has no meaning for them.
If such column added users should be able to hide/show it as a system setting (or maybe preferably as a user setting as it could be that in the same Airflow instance one team works only in UTC while other teams combine timezones).
